solutions/swb-app/src/validatorHelper.ts

```diff
@@ -21,4 +22,22 @@ function processValidatorResult(validatorResult: ValidatorResult): void {
   }
 }
 
-export { processValidatorResult };
+function validateAndParse<T>(parser: ZodTypeAny, data: unknown): T {
```

Comment:
This was copied over from `feat/accounts-api` branch

Reply:
can we have this in develop ?

Reply:
Yep, I'll merge all this code to `feat/costCenter`, then I'll pull out the common code and create a PR to merge to `develop`